source/index.html.md.erb

```diff
@@ -27,7 +27,7 @@ search: false
 <% HEADER_AUTH = "**Authorization**: Bearer ll352u9jujauoqz4gstvsae05" %>
 <% HEADER_CT_JSON = "**Content-Type**: application/json" %>
 <% PARAMETER_ALLOW_PARTIAL_SUCCESS = "**allowPartialSuccess** (optional): when specified with a value of `true`, enables partial success for this bulk operation.  See [Partial Success](#partial-success) for more information." %>
-<% PARAMETER_MODIFIEDSINCE = "**modifiedSince** (optional): when specified with a <a href='#dates-and-times'>date and time</a> value, response only includes the objects that are modified on or after the date and time specified." %>
+<% PARAMETER_MODIFIEDSINCE = "**modifiedSince** (optional): when specified with a <a href='#dates-and-times'>date and time</a> value, response only includes the objects that are modified on or after the date and time specified. The return value is only accurate within 1-2 hours because of lag time. If you need more precise information about updates, use [Get Sheet Version](#get-sheet-version)." %>
```

Comment:
Let's refine this text

Reply:
How about something more like this: "If you need to keep track of frequent changes, it may be more useful to  use "Get Sheet Version."

Reply:
Done.